src/sagas.ts

```diff
@@ -0,0 +1,44 @@
+import { setHeroesError, setCharacters, setIsLoading as setIsHeroesLoading, GET_HEROES } from './actions/search';
+import { Action } from 'redux';
+import { call, put, takeLatest } from 'redux-saga/effects';
+import { handleRequest } from './api/requestHandler';
+import { GET_COMICS, setComics, setComicsError, setIsLoading as setIsComicsLoading  } from './actions/comics';
+
+interface ITask {
+  payload: string;
+}
+
+interface TaskAction extends Action, ITask {
+  type: 'TASK_ADD';
+}
+
+function* fetchHeroes(action: { payload: string }): any {
+  try {
+    yield put(setIsHeroesLoading(true));
+    const heroes = yield call(<any>handleRequest, `characters`, action.payload);
```

Comment:
any

Reply:
there are issues with saga's typings and I'm not sure how to resolve them. Could you please give some advice?